client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -185,6 +188,12 @@ class ListItem extends PureComponent {
     );
   };
 
+  preventDefault = event => preventDefault(event);
```

Comment:
is this additional wrapper for preventDefault necessary? Where this method is used in this component?